scripts/testnet/variables.sh

```diff
@@ -49,15 +49,15 @@ export TX_SIGN_FORMAT="bech32"
 export BLOCK_SIGN_FORMAT="hex"
 
 # Shard structure
-export SHARDCOUNT=2
-export SHARD_VALIDATORCOUNT=3
+export SHARDCOUNT=1
```

Comment:
Do we want to modify the default values from this script?

Reply:
nope, updone